agent/build.gradle

```diff
@@ -19,6 +19,9 @@ ext {
 }
 
 dependencies {
+	implementation project(':teamscale-client')
+	implementation project(':report-generator')
+
```

Comment:
I reformatted the code again but no changes. Either I have to configure my project properly or we decide on globally configured formatting settings for this the `teamscale-jacoco-agent` project and ensure it. In the latter case I'd suggest new PR otherwise it'd be great if we could have a look at my configuration together.